src/main/java/com/jagrosh/jdautilities/menu/pagination/PaginatorBuilder.java

```diff
@@ -164,10 +201,10 @@ public PaginatorBuilder useNumberedItems(boolean number)
     }
     
     /**
-     * Sets whether or not the paginator should wait for an input when
-     * only one page is visible
-     * @param wait if the paginator should wait
-     * @return the builder
+     * This method honestly makes no sense...
```

Comment:
That shouldn't have survived, I believe that was fixed in an edit, I must have forgotten to update it.